client/js/JSUtils.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+/*
+ * JSUtils.js
+ * Copyright(c) 2016 Universidad de Las Palmas de Gran Canaria
+ * Authors:
+ *   Jaisiel Santana <jaisiel@gmail.com>
+ *   Alejandro Sánchez <alemagox@gmail.com>
+ *   Pablo Fernández <pablo.fernandez@ulpgc.es>
+ * MIT Licensed
+
+*/
+
+var Tourguide = Tourguide || {};
```

Comment:
I can see the good intentions on having a root module called TourGuide, but it is making the code difficult to read and error prone as you depend on the previous creation of an object named 'TourGuide' . Why don't you just create a module named Request 

```js
var Request = (function() {
   return { 
      'get': getFunction,
      'post': postFunction 
   }
})();
```